src/lib/detection_algorithms/main.js

```diff
@@ -0,0 +1,50 @@
+//read from config to find detection algorthims and paths
+const path = require('path');
+const config_path = path.join(__dirname, 'config.json')
+const config = require(config_path);
+
+//load detection modules
+var detection_modules = {}
+Object.keys(config).forEach((module_path) => {
+    detection_modules[config[module_path]] =  require(path.join(__dirname, module_path));
+});
+
+function DetectionAlgorithmManager() {
+    var exporter, reviewer, export_algorithm, review_algorthim;
+    //this returns a list of the availble detection modules
+    this.getAvailbleAlgorthims = function() {
+        return config;
+    },
+
+    //returns export detection algorithm
+    this.getCurrentExportAlgorithm = function() {
+        return export_algorithm;
+    },
+
+    //returns current review algorithm
+    this.getCurrentExportAlgorithm = function() {
+        return review_algorthim;
+    },
+
+
+    //Set the  exporter to the specified detection module
+    this.setExporter = function(algorithm, exportDirPath, classes, posFramesCount, frameWidth, frameHeight, testSplit) {
+         if (!Object.values(config).includes(algorithm)){
+             throw (`Error ${algorithm} module is not recognized`);
+         }
+         this.exporter = new detection_modules[algorithm].Exporter(exportDirPath, classes, posFramesCount, frameWidth, frameHeight, testSplit);
```

Comment:
Don't export the reviewer and exporter directly - wrap them with function.
It might maker more sense to get rid of setExporter and setReviewer altogether and just have export and review functions that accept the algorithm name.